### PR TITLE
Add functionality to push a Borrel Reservation to Tantalus

### PR DESCRIPTION
**Pull request by KiOui**
Add functionality to push a Borrel Reservation to Tantalus.

This PR (yes I know it's not nice to do all these things at once):
- Updates the dependencies
- Fixes an `orders` test
- Adds functionality to push a Borrel Reservation to Tantalus.

### CODE REVIEW

**Comment by KiOui** [comment]
Things left to do:

- [ ] Add a PR on [Tantalus](https://github.com/thijsmie/tantalus) to create an endpoint which TOSTI can use to synchronize Borrel Reservations to Tantalus.
- [ ] Add code to the `synchronize_borrelreservation_to_tantalus` function to push the Borrel Reservation to Tantalus.

**Comment by KiOui** [comment]
I would suggest checking the code and the tests and not setting up a dev environment to test the sync to Tantalus itself. The Tantalus PR (that is already merged) can be found [here](https://github.com/thijsmie/tantalus/pull/30).

File: website/tantalus/admin.py [hunk 1]
```diff
@@ -59,7 +88,7 @@ def changeform_view(self, request, object_id=None, form_url="", extra_context=No
             obj = None
 
         if self._should_do_push_to_tantalus(obj, request):
-            if synchronize_to_tantalus(obj):
+            if synchronize_shift_to_tantalus(obj):
                 self.message_user(request, format_html("Tantalus synchronisation succeeded."), messages.SUCCESS)
                 if not TantalusShiftSynchronization.objects.filter(shift=obj).exists():
```
Comment (JobDoesburg) [inline website/tantalus/admin.py:93]: Nicer is to use `TantalusShiftSynchronization.objects.get_or_create`

**Review by JobDoesburg — APPROVED** [review]
This really needs https://github.com/JobDoesburg/django-easy-admin-object-actions

File: website/tantalus/templates/admin/borrel/borrelreservation/submit_line.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    {% url opts|admin_urlname:'delete' original.pk|admin_urlquote as delete_url %}
+    <p class="deletelink-box"><a href="{% add_preserved_filters delete_url %}" class="deletelink">Delete</a></p>
+{% endif %}
+{% if show_push_to_tantalus %}
+    <input type="submit" value="Push to Tantalus" name="_pushtotantalus" id="push-to-tantalus">
+{% endif %}
+{% if show_force_push_to_tantalus %}
+    <input type="submit" value="Force push to Tantalus" name="_pushtotantalus" id="force-push-to-tantalus">
+    <script>
+        const force_push_button = document.getElementById('force-push-to-tantalus');
```
Comment (Gulianrdgd) [inline website/tantalus/templates/admin/borrel/borrelreservation/submit_line.html:16]: Is a force push option really needed or will it only create problems when force pushing? I mean if there is a problem the treasurer can simply add the order manually right?

**Review by Gulianrdgd — APPROVED** [review]

File: website/tantalus/templates/admin/borrel/borrelreservation/submit_line.html (hunk 2 above)
Comment (KiOui) [reply website/tantalus/templates/admin/borrel/borrelreservation/submit_line.html:16]: Good point! I'll ask Douwe in the break!

Comment (JobDoesburg) [reply website/tantalus/templates/admin/borrel/borrelreservation/submit_line.html:16]: I will rewrite to https://github.com/JobDoesburg/django-easy-admin-object-actions soon ™   and then we will remove large parts of this code again anyhow

**Pull request merged**